synapse/handlers/register.py

```diff
@@ -26,6 +26,7 @@
     RegistrationError,
     SynapseError,
 )
+from synapse.config._base import ConfigError
```

Comment:
can you import this from `synapse.config` rather than the private `_base` module, please? (not that it's really appropriate to be raising ConfigErrors at runtime imho)